src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -74,26 +76,39 @@ private static DeviceDataManager createDeviceDataManager() {
 	}
 
 	/**
-	 * Creates a data model with 3 devices.
-	 * All are at max_tx_power, which represents the first step in greedy TPC.
+	 * Creates a data model with 3 devices. All are at max_tx_power, which
+	 * represents the first step in greedy TPC.
+	 *
+	 * @param deviceCIn5G true if device C should operate in the 5G band
+	 * @return a data model
 	 */
-	private static DataModel createModel() {
+	private static DataModel createModel(boolean deviceCIn5G) {
```

Comment:
This is a very odd function signature and makes it hard to read/understand the tests which call this. For example you could have 3 parameters where you pass in the band name (constant) for each device. Or take no arguments and modify the result (`DataModel`) afterwards where needed.

Reply:
I will modify the result afterwards in the only place where `false` is passed in and keep the `true`-passed-in behavior the default.